dataactbroker/handlers/userHandler.py

```diff
@@ -390,7 +390,7 @@ def getPermssionList(self):
         queryResult = self.session.query(PermissionType).all()
         return queryResult
 
-    def createUserWithPassword(self,email,password,bcrypt,admin=False):
+    def createUserWithPassword(self,email,password,bcrypt,permission=1):
```

Comment:
Does the value of `permission` represent `permission_type_id` in the permission_type_table? I'm still a bit unclear on how the broker sets permissions, but this smells like a magic number. We have a few of those in the code base already (e.g., in unit tests), but we should avoid introducing more.